Deploy/kubernetes/jobservice/jobservice.rc.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: jobservice-rc
+  namespace: harbor
+  labels:
+    name: jobservice-rc
+spec:
+  replicas: 1
+  selector:
+    name: jobservice-apps
+  template:
+    metadata:
+      namespace: harbor
+      labels:
+        name: jobservice-apps
+    spec:
+      containers:
+      - name: jobservice-app
+        image: test.caicloud.io/harbor/jobservice:0.4.0
```

Comment:
it won't work,need i push these images to offcial registry of docker or a common registry?